var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -97,13 +98,14 @@ class Apex(CMakePackage):
     variant("lmsensors", default=False, description="Enables LM-Sensors support")
     variant("mpi", default=False, description="Enables MPI support")
     variant("starpu", default=False, description="Enables StarPU support")
+    variant("opencl", default=False, description="Enables OpenCL support")
     variant("tests", default=False, description="Build Unit Tests")
     variant("examples", default=False, description="Build Examples")
 
     # Dependencies
     depends_on("zlib-api")
-    depends_on("cmake@3.10.0:", type="build")
-    depends_on("kokkos", type="build", when="+kokkos")
+    depends_on("cmake@3.20.1:", type="build")
```

Comment:
Should this be
```suggestion
    depends_on("cmake@3.10.0:", type="build")
    depends_on("cmake@3.20.1:", type="build", when="@2.6.2:")
```
(based on https://github.com/UO-OACISS/apex/commit/96d713887fd1f7d17bb336801e46e877ccf2d2e0) or was 3.20.1 anyway effectively required for older versions?

Reply:
Yeah, I that's a better change, too.